src/dracula.yml

```diff
@@ -1131,4 +1131,51 @@ tokenColors:
       - log.warning
     settings:
       foreground: *YELLOW
-      fontStyle: bold
\ No newline at end of file
+      fontStyle: bold
+
+  - name: C++ Namespace
```

Comment:
For maintainability, I'd prefer to not have c++ specific rules.

Instead, you should look for existing rules that match what you're looking for and add the selectors to those.

Reply:
I have made changes requested. This pr started because I couldn't find existing rules for c++